pom.xml

```diff
@@ -8,6 +8,40 @@
   <artifactId>docopt</artifactId>
   <version>0.6.0-SNAPSHOT</version>
 
+  <name>docopt.java</name>
+  <url>https://github.com/docopt/docopt.java</url>
+  <description>Java port of docopt</description>
+
+  <parent>
```

Comment:
This method has been deprecated because it pulls in any information that isn't overridden here. Best practice now is to copy/paste the repository and profile from oss-parent into this pom file.